components/shared/SquareButton.js

```diff
@@ -3,23 +3,28 @@ import styled from 'styled-components';
 import Icon from './Icon';
 import * as gtag from '../../lib/gtag';
 
-const SquareButton = ({ className, icon, iconClass, onClick }) => {
+const SquareButton = props => {
```

Comment:
You can spread props at the end of a destructuring, so I generally like to leave this as `({ className, icon, iconClass, onClick, ...props })`

Then, down below, you can spread it back into your lower level component to handle passing anything else down.